PowerUp/app/src/main/java/powerup/systers/com/StartActivity.java

```diff
@@ -33,7 +37,26 @@ protected void onCreate(Bundle savedInstanceState) {
         newUserButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                startActivityForResult(new Intent(StartActivity.this, AvatarRoomActivity.class), 0);
+                if (hasPreviouslyStarted) {
+                    new AlertDialog.Builder(StartActivity.this)
+                            .setTitle("New Game")
+                            .setMessage("Wish to a start new game? All your saved progress might be lost.")
+                            .setPositiveButton(getResources().getString(R.string.yes), new DialogInterface.OnClickListener() {
+                                public void onClick(DialogInterface dialog, int which) {
+                                    // continue with delete
```

Comment:
oh I am sorry about that. Should have paid attention.
Give me a minute will correct it.